package.json

```diff
@@ -118,5 +117,9 @@
 		"minimist": "^1.2.6"
 	},
 	"prettier": "@sapphire/prettier-config",
-	"packageManager": "yarn@3.2.1"
+	"packageManager": "yarn@3.2.1",
+	"dependencies": {
```

Comment:
Why are you reverting this?

Reply:
why do we need to bundle the deps for nodejs users? they can always install it.

Reply:
Makes our lives easier when bundling to have consistency

Reply:
I disagree. Dependencies should be installed by end-users for CJS / ESM. Not to mention that `fast-deep-equal` is already likely a dependency of 9/10 projects anyway as per how it is required by other packages:

<img width="517" alt="image" src="https://user-images.githubusercontent.com/4019718/178068832-27270c42-1439-4c6a-af4b-fc8823935bff.png">